docs/tutorials/complete_applications/scenario_management_overview/step_06/ReadMe.md

```diff
@@ -94,12 +74,12 @@ Results:
 
 ```
 Scenario 1: submit
-[2022-12-22 16:20:09,079][Taipy][INFO] job JOB_filter_by_month_0d7836eb-70eb-4fe6-b954-0e56967831b6 is completed.
-[2022-12-22 16:20:09,177][Taipy][INFO] job JOB_count_values_91214241-ce81-42d8-9025-e83509652133 is completed.
+[2022-12-22 16:20:09,079][Taipy][INFO] job JOB_filter_by_month_... is completed.
+[2022-12-22 16:20:09,177][Taipy][INFO] job JOB_count_values_... is completed.
 Value 849
 ```
 
-When submitting scenario 2, the scheduler will skip the first task of this second scenario. Indeed, the two scenarios share the same input Data Nodes for this task, and no changes have occurred on these Data Nodes (since the last task run when we submitted scenario 1).
+When submitting scenario 2, the scheduler skips the first task of this second scenario. Indeed, the two scenarios share the same Data Nodes for this task, and no changes have occurred on the input Data Nodes (since the last task run when we submitted scenario 1).
```

Comment:
```suggestion
When you submit scenario 2, the scheduler skips the initial task in this second scenario. This is because both scenarios use the same Data Nodes for this task, and there have been no changes in the input Data Nodes since the last time the task ran when we submitted scenario 1.
```